utils/docker.go

```diff
@@ -0,0 +1,80 @@
+package utils
```

Comment:
In the penumbra PR #33 , I added another util file `file.go` for a file `Copy` helper. So maybe it should be `utils/docker` with one file `dockerutils.go` with package name `dockerutils`, and then `utils/file` with one file `fileutils.go` with package name `fileutils`? cc @mark-rushakoff since he brought up the same concern.